### PR TITLE
Add page for react-native-sketch-canvas component

### PR DESCRIPTION
**Pull request by tritao**
ApplicationFrameHost_eUCQfnEzw5

### CODE REVIEW

File: src/RNGalleryList.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {TimePickerExamplePage} from './examples/TimePickerExamplePage';
 import {SliderExamplePage} from './examples/SliderExamplePage';
 import {PickerExamplePage} from './examples/PickerExamplePage';
 import {DeviceInfoExamplePage} from './examples/DeviceInfoExamplePage';
+import { SketchExamplePage } from './examples/SketchExamplePage';
```
Comment (chiaramooney) [inline src/RNGalleryList.ts:11]: nit: extra spaces around 'SketchExamplePage'

File: src/examples/SketchExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+'use strict';
+import {Button, Linking, StyleSheet, Text, View} from 'react-native';
+import React, { useRef } from 'react';
```
Comment (chiaramooney) [inline src/examples/SketchExamplePage.tsx:3]: nit: extra spaces around 'useRef'

File: src/examples/SketchExamplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+
+import RNSketchCanvas, { SketchCanvas } from '@terrylinla/react-native-sketch-canvas';
```
Comment (chiaramooney) [inline src/examples/SketchExamplePage.tsx:7]: nit: extra new line, extra spaces around 'SketchCanvas'

File: src/examples/SketchExamplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <View style={[{ backgroundColor: color, borderWidth: 2 }, styles.strokeColorButton]} />
+  );
+
+  return (
+    <Page
+      title="Sketch Canvas"
+      description="Shows an example of the react-native-sketch-canvas component for sketching.">
+
+      <View >
+        <Text>For more information about the supported APIs, check out the{' '}
```
Comment (chiaramooney) [inline src/examples/SketchExamplePage.tsx:51]: PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page in replacement of this?

File: src/examples/SketchExamplePage.tsx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+          <Text style={{color: 'blue'}} onPress={() =>
+             Linking.openURL('https://github.com/creambyemute/react-native-sketch-canvas#usage')}>
+          documentation
+          </Text>
+        .
+        </Text>
+      </View>
+
+      <Example title="" code={exampleJsx}>
```
Comment (chiaramooney) [inline src/examples/SketchExamplePage.tsx:60]: nit: add title here "A simple Sketch Canvas."

**Comment by tritao** [comment]
Updated.

**Review by chiaramooney — APPROVED** [review]
👍 looks great!

**Comment by chiaramooney** [comment]
@tritao Looks like there are some merge conflicts that need to be resolved before I can merge your changes. Could you resolve those?

**Comment by tritao** [comment]
OK, I've rebased this one on top of main, and https://github.com/microsoft/react-native-gallery/pull/27 on top of this one.

Please merge these two first, and then I'll rebase the rest, to minimize the chance of more conflicts.

**Pull request merged**